### PR TITLE
Util modification - just an excuse to create a release commit

### PR DESCRIPTION
**Pull request by bemailloux**
Oops I forgot this repo was managed by semantic-versioning.

### CODE REVIEW

**Review by NicholasHallman — APPROVED** [review]

File: demo/util.js [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-		newNodes.push(createNode(newNodeId, [node.Id], node.Type + 1));
+		newNodes.push(createNode(++idCounter, [node.Id], node.Type + 1));
 	}
 	return newNodes;
 }
 
-// id function may not work for n > 9
 export function createNaryTree(n, numNodes) {
 
+	idCounter = 1;
```
Comment (anhill-D2L) [inline demo/util.js:19]: Will resetting the idCounter mean that all of our nodes will have id's from 1-n?

**Comment by bemailloux** [comment]
> Don't you need something in your commit message for semver to pick up?

Yeah. That goes in the merge commit message, doesn't it?

**Review by anhill-D2L — APPROVED** [review]

**Comment by johngwilkinson** [comment]
> > Don't you need something in your commit message for semver to pick up?
> 
> Yeah. That goes in the merge commit message, doesn't it?

Does github ui let you edit the merge commit message? I think the usual practice is to tag each commit with what kind of thing it is, so semver knows what to do with the merge commit as a whole. I kind of inferred that myself though - could be wrong.

**Comment by bemailloux** [comment]
> > > Don't you need something in your commit message for semver to pick up?
> > 
> > 
> > Yeah. That goes in the merge commit message, doesn't it?
> 
> Does github ui let you edit the merge commit message? I think the usual practice is to tag each commit with what kind of thing it is, so semver knows what to do with the merge commit as a whole. I kind of inferred that myself though - could be wrong.

It does, but I've managed to mess it up again (edited the commit description instead of the message itself 😠 )

it's almost like we need automation for our automation

**Pull request merged**